lib/ssl/doc/src/ssl.xml

```diff
@@ -299,6 +303,15 @@
       and to restrict their usage when using a cipher suite supporting them.
       </p></item>
 
+      <tag><c>{supported_groups, [group()]}</c></tag>
+      <item><p> Used only in TLS 1.3 and obsoletes the eccs option. It indicates the named
+      groups which the client supports for key exchange, ordered from most preferred to least
+      preferred. In case of a server it is used to update the client's view of its preferences.</p>
+      <warning><p>Note that in versions of TLS prior to TLS 1.3, this option only contained
```

Comment:
This did not feel entirely clear to me.  Maybe say that the obsoleted eccs option was also used for selection of elliptic curves for signature purposes as well as key exchange before TLS-1.3, but  TLS-1.3 moves that functionality for signatures to signature_algs and signature_algs_cert extensions.

Reply:
Will fix.